code/modules/blob/blob.dm

```diff
@@ -4,16 +4,21 @@
 	icon = 'icons/mob/blob.dmi'
 	icon_state = "blob"
 	light_range = 3
+	light_color = "#b5ff5b"
 	desc = "Some blob creature thingy"
 	density = 1
 	opacity = 0
 	anchored = 1
 	mouse_opacity = 2
+	layer = 4
 
 	var/maxHealth = 30
 	var/health
+	var/regen_rate = 5
 	var/brute_resist = 4
+	var/laser_resist = 5	// Special resist for laser based weapons - Emitters or handheld energy weaponry. Damage is divided by this and THEN by fire_resist.
```

Comment:
i approve of laser resistance but this seems like a crazy amount. This amounts to 80% damage reduction from lasers. 
They ALSO have 75% reduction from brute damage, so what are you actually supposed to hit them with?
At this point you might as well just increase their health

I agree blobs need a buff, but i don't think making them this much more powerful is a good idea.

Whats the design goal here? How do you want crew to fight blobs?